src/core_plugins/inspector_views/public/data/data_table.less

```diff
@@ -0,0 +1,7 @@
+.inspector-table__filter {
+  opacity: 0;
```

Comment:
Don't forget to up the opacity back when focused for acessibility. 

![image](https://user-images.githubusercontent.com/324519/40748891-caa89c90-6416-11e8-8e8f-a72a66508cd4.png)

Reply:
Done

Reply:
.. and very embarrassing!